Formula/cilium-cli.rb

```diff
@@ -0,0 +1,26 @@
+class CiliumCli < Formula
+  desc "CLI to install, manage & troubleshoot Kubernetes clusters running Cilium"
+  homepage "https://cilium.io"
+  url "https://github.com/cilium/cilium-cli/archive/refs/tags/v0.8.6.tar.gz"
+  sha256 "7d4adfc511206af56570ba90dd523bd31f1703c8f9b36bca90e4d0863dd06ab7"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X github.com/cilium/cilium-cli/internal/cli/cmd.Version=#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/cilium", "./cmd/cilium"
+
+    bash_output = Utils.safe_popen_read(bin/"cilium", "completion", "bash")
+    (bash_completion/"cilium").write bash_output
+    zsh_output = Utils.safe_popen_read(bin/"cilium", "completion", "zsh")
+    (zsh_completion/"_cilium").write zsh_output
+    fish_output = Utils.safe_popen_read(bin/"cilium", "completion", "fish")
+    (fish_completion/"cilium.fish").write fish_output
+  end
+
+  test do
+    assert_match(/Cluster name "" is not valid/, shell_output("#{bin}/cilium install 2>&1", 1))
+    assert_match(/Error: Unable to enable Hubble/, shell_output("#{bin}/cilium hubble enable 2>&1", 1))
```

Comment:
```suggestion
    assert_match('Cluster name "" is not valid', shell_output("#{bin}/cilium install 2>&1", 1))
    assert_match("Error: Unable to enable Hubble", shell_output("#{bin}/cilium hubble enable 2>&1", 1))
```
We don't need these to be `Regexp`s.

Reply:
Done, thanks.

Reply:
You actually forgot to push this change, I think.

Reply:
Indeed, thanks. Now pushed.